docs/conf.py

```diff
@@ -125,6 +125,14 @@
     # Setup.py files of the providers
     "_api/airflow/providers/*/*setup/index.rst",
     "_api/airflow/providers/*/*/*setup/index.rst",
+    # Other API rst files.
+    "_api/endpoints/index.rst",
+    "_api/endpoints/*/index.rst",
+    "_api/exceptions/index.rst",
+    "_api/parameters/index.rst",
+    "_api/schemas/index.rst",
+    "_api/schemas/*/index.rst",
+    "_api/security/index.rst",
```

Comment:
Something here looks suspect -- these are files form with `airflow/api_connexion/` -- how is the `api_connexion` path getting stripped/lost?

Reply:
I have no idea, but the doc building failed without those. This is what I discussed with @mik-laj and that was the advice I had from him. I would love to know where it comes from, but sphinx and it's error messages for AutoAPI often are not really useful. @mik-laj -> do you think those API links are removed legitimately or should we find a reason why they are not being generated?

Reply:
Spoke to @mik-laj -> he confirmed they should be ignored now. They are not used/needed (documentation about API we got via swagger) simply autoapi does not ignore those automatically now due to some configuration problem that Kamil will fix when he will fix provider docs/split them.

Reply:
I will create tickets describing my improvements in documentation, but I am waiting for the idea to be accepted to quit RTD and to divide documentation for each provider. 
https://github.com/apache/airflow/issues/11423
when the documentation is multi-part it will be easier to fix all problems.

Reply:
My issue is not that we are ignoring these files, but that the paths look wrong.

For instance I would expect `_api/api_connextion/security/index.rst` NOT `_api/security/index.rst`